samples/sampler/stories/default/TooltipDecorator.js

```diff
@@ -1,30 +1,62 @@
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import {action} from '@enact/storybook-utils/addons/actions';
-import {boolean, select, text} from '@enact/storybook-utils/addons/knobs';
+import {boolean, number, select, text} from '@enact/storybook-utils/addons/knobs';
 import {storiesOf} from '@storybook/react';
-import Button, {ButtonBase} from '@enact/agate/Button';
-import TooltipDecorator from '@enact/agate/TooltipDecorator';
-import UiButton from '@enact/ui/Button';
+import Button from '@enact/agate/Button';
+import TooltipDecorator, {Tooltip, TooltipBase} from '@enact/agate/TooltipDecorator';
 import React from 'react';
 
+import iconNames from './icons';
+
 Button.displayName = 'TooltipDecorator';
-const Config = mergeComponentMetadata('TooltipDecorator', UiButton, ButtonBase, Button);
-const TooltipButton = TooltipDecorator({tooltipDestinationProp: 'decoration'}, Button);
+const Config = mergeComponentMetadata('TooltipDecorator',  TooltipDecorator, Tooltip, TooltipBase);
+const TooltipButton = TooltipDecorator(
+	{tooltipDestinationProp: 'decoration'},
+	Button
+);
+
+const prop = {
+	icons: ['', ...iconNames],
+	tooltipPosition: [
+		'above',
+		'above center',
+		'above left',
+		'above right',
+		'below',
+		'below center',
+		'below left',
+		'below right',
+		'left bottom',
+		'left middle',
+		'left top',
+		'right bottom',
+		'right middle',
+		'right top'
+	]
+};
 
 storiesOf('Agate', module)
 	.add(
 		'TooltipDecorator',
 		() => (
-			<TooltipButton
-				backgroundOpacity={select('backgroundOpacity', ['opaque', 'lightOpaque', 'transparent'], Config)}
-				disabled={boolean('disabled', Config)}
-				onClick={action('onClick')}
-				selected={boolean('selected', Config)}
-				size={select('size', ['smallest', 'small', 'large', 'huge'], Config)}
-				tooltipText={text('tooltip', Config, 'Tooltip')}
-			>
-				{text('children', Button, 'Click me')}
-			</TooltipButton>
+			<div style={{textAlign: 'center'}}>
+				<TooltipButton
+					backgroundOpacity={select('backgroundOpacity', ['opaque', 'lightOpaque', 'transparent'], Config)}
+					disabled={boolean('disabled', Config)}
+					icon={select('icon', prop.icons, Config)}
+					onClick={action('onClick')}
+					selected={boolean('selected', Config)}
+					size={select('size', ['smallest', 'small', 'large', 'huge'], Config)}
+					tooltipDelay={number('tooltipDelay', Config, 500)}
+					tooltipMarquee={boolean('tooltipMarquee', Config)}
+					tooltipRelative={boolean('tooltipRelative', Config)}
+					tooltipPosition={select('tooltipPosition', prop.tooltipPosition, Config)}
+					tooltipText={text('tooltip', Config, 'Tooltip')}
+					tooltipWidth={number('tooltipWidth', Config)}
```

Comment:
There is console error when I enter or remove this knob value.
react_devtools_backend.js:2430 Warning: `value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.

Reply:
I noticed we encountered this problem on every story that uses a number type knob (both in Agate and Sandstone). I tested it on Nebula environment and I got no warnings, but locally it does reproduce. We could look more into this on a separate task since it concerns more that the Tooltip component. If you have any idea why it doesn't reproduce on Nebula please let us know.

Reply:
OK. Please make Jira issue and leave the PLAT number on this PR description using "Additional Considerations"